environment.yaml

```diff
@@ -0,0 +1,25 @@
+name: dropsonde_test 
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python>=3.10
```

Comment:
From own experience, it is better so set an explicit Python environment. Otherwise it might depend on the base conda/mamba/micromamba version which will cause confusion at some point. I would go for 3.12